bedrock/settings/appstores.py

```diff
@@ -85,3 +85,16 @@
     'vi': 'vt',     # Vietnam
     'en-ZA': 'za',  # South Africa
 }
+
+# Link to Firefox Focus for iOS on the Apple App Store via app.adjust.
+# Fallback parameter sends users who are on the wrong platform to the iTunes website.
+# https://bugzilla.mozilla.org/show_bug.cgi?id=1350170
+APPLE_APPSTORE_FIREFOX_FOCUS_LINK = ('https://app.adjust.com/b8s7qo?campaign=&adgroup=Focus&creative=iOS&' +
```

Comment:
Nice catch - I think you're right. I'm confirming with mhan in the bug, but getting two fully formed URLs pasted there is currently the biggest challenge for this page it seems 😩.